app/services/designation-service.js

```diff
@@ -25,14 +27,16 @@ function ID(modelOrId) {
  * <br> which mostly involves operations on the orders_package join table
  *
  */
-export default ApiBaseService.extend(NavigationAwareness, {
+export default ApiBaseService.extend(NavigationAwareness, AsyncMixin, {
   store: Ember.inject.service(),
 
   init() {
     this._super(...arguments);
     this.set("openOrderSearch", false);
   },
 
+  editableQty: Ember.computed.alias("settings.allowPartialOperations"),
+
```

Comment:
I'm wondering if those are needed now that we've retired stockit, not a big deal to have it though